TKGame/Music.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Media;//Allows the use of MediaPlayer
+
+
+namespace TKGame
+{
+    public class Music
+    {
+        #region Member Variables
+        private static Song backgroundSong;
+        #endregion
+
+        /// <summary>
+        /// Loads up audio and plays it. Takes a Content Manager and a float volume value.
+        /// </summary>
+        /// <param name="Content"></param>
+        /// <param name="vol"></param>
+        public static void LoadContent(ContentManager Content, float vol)
```

Comment:
We probably want to be able to change the volume with a function call or something as well

Reply:
You guys don't have volume controls on your keyboard???